client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -53,53 +53,53 @@ class MemberProfilePage extends Component {
         facebook: "www.facebook.com",
         instagram: "www.instagram.com",
         pinterest: "www.pinterest.com",
-        isCurrentUser: false
+        isCurrentUser: false,
       },
-      { name: "Martha", bio: "sup yall" }
+      { name: "Martha", bio: "sup yall" },
     ];
     let testUsers = [
       {
         userId: 1,
         userFirstName: "Paula",
         userSurname: "Bannerman",
-        userPic: pic1
+        userPic: pic1,
```

Comment:
If they're user objects, then the properties don't need `user` in them

Reply:
Not sure why this was added.  The only change to the js file was adding one additional div + className.